jenkins/github_helpers.py

```diff
@@ -56,6 +58,8 @@ def get_github_instance(self):
     def get_github_token(self):
         return self.github_token
 
+    # FIXME: Probably can end up picking repo from wrong org if two
+    # repos have the same name in different orgs.
     def connect_to_repo(self, github_instance, repo_name):
```

Comment:
1. Is this still used, or should it be deleted?
2. If it needs to stay, do you want it to be fixed, or should it be marked as deprecated, with an explanation of why it is deprecated and how to use `repo_from_remote` instead?

Reply:
It's still used, and I want it gone. I like the idea of pointing out `repo_from_remote` as a better option in the interim.